workspaces/bulk-import/plugins/bulk-import-backend/package.json

```
@@ -43,8 +43,11 @@
     "openapi": "./scripts/openapi.sh"
   },
   "dependencies": {
+    "@ai-zen/node-fetch-event-source": "2.1.4",
```

Is there a reason that we use this fork instead of

```suggestion
    "@microsoft/fetch-event-source": "^2.0.1",
```

I saw that https://www.npmjs.com/package/@ai-zen/node-fetch-event-source refers this https://github.com/ai-zen/node-fetch-event-source which is a 5 year old fork of https://github.com/Azure/fetch-event-source

And actually the one from the microsoft org is what Backstage is using.

I guess at the end you might could use `ScaffolderClient` from @backstage/scaffolder-common directly.

For now I would suggest switch to the microsoft dependency and creating a bug to switch to ScaffolderClient as a follow-up.

Wdyt @AndrienkoAleksandr @gashcrumb ?